test/features/login_redirect.feature

```diff
@@ -0,0 +1,55 @@
+@login_redirect
+Feature: Login Redirection
+
+    @dashboard_login
+    Scenario: As an unauthenticated user, when I visit the /dashboard URL I see the login page
+        Given "Unathenticated" as the persona
+        When I visit "/dashboard"
+        Then I should see an element with xpath "//h1[contains(string(), 'Login')]"
```

Comment:
are we able to test the url suffix with:
And I should be on url ending with "/login/?came_from?/dashboard"  
as an example.

Reply:
For that particular test, the default return location after logging in is the dashboard, so don't set the `came_from` parameter in the request.